CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Internal
 
+- BC-Break: Changed - renamed namespace [#10](https://github.com/php-parallel-lint/PHP-Console-Color/pull/10) [@grogy](https://github.com/grogy).
```

Comment:
I'd suggest moving the changelog entry up into a `### Changed` header. This is **_not_** an _internal_ change, but a change to the Public API, so should be annotated as such.

Reply:
You have the right